plop-templates/Component.theme.ts.hbs

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
Please don't add a command to add the `theme.ts` file to the component. We should store only the global styles and variables in `theme`, not style objects from all of the components.